packages/openapi-generator/src/wrapper-files/index-file.ts

```diff
@@ -1,13 +1,21 @@
-import { codeBlock } from '@sap-cloud-sdk/util';
+import { codeBlock, first } from '@sap-cloud-sdk/util';
 
 /**
  * @experimental This API is experimental and might change in newer versions. Use with caution.
  * Generate the index file to expose the SAP Cloud SDK API and models.
+ * @param apis The generated api files.
  * @returns The index file contents.
  */
-export function indexFile(): string {
+export function indexFile(apis: string[]): string {
   return codeBlock`
 export * from './openapi/model';
-export * from './api';
+${apis
+  .filter(api => api.endsWith('.ts'))
+  .map(api => exportAll(first(api.split('.ts'))!))
```

Comment:
The first(api.split('.ts')) is to remove the `.ts` on the filename right? This seems a bit hacky. I would make the exportAll method a bit more intelligent and allow as input `file` and `file.ts` and do a `replace(/$\.ts/,"") on it so that it is clear what is done.

=> question what would happen if someone would have a tag `ts.LaLA`. Is this not allowed? In general we should check that our file generation also works for that odd value.

Reply:
I'll create a test case for this.

Reply:
Also cosider my comment about the `split`. A replace is closer to the thing really done.

Reply:
The split is not needed any more, I removed it.